### PR TITLE
Added a query param to force the location.

### PR DESCRIPTION
**Pull request by nickoneill**
This is coming in handy with automating something I'm working on, but could also be used to send people to the right districts if they're split and don't know how to fix.

### CODE REVIEW

File: src/redux/remoteData/asyncActionCreator.ts [hunk 1]
```diff
@@ -211,6 +211,19 @@ export const startup = () => {
...
     dispatch(clearContactIndexes());
 
+    // if a location is passed as a query, override or set the location address manually
+    let addressQuery = 'forceAddress';
+    let query = window.location.search.substring(1);
+    let vars = query.split('&');
+    for (let i = 0; i < vars.length; i++) {
+        let pair = vars[i].split('=');
+        if (decodeURIComponent(pair[0]) === addressQuery) {
+          state.locationState.address = pair[1];
```
Comment (cdoremus) [inline src/redux/remoteData/asyncActionCreator.ts]: State always needs to be updated by using a Redux reducer via an Action created by an ActionCreator.

**Review by cdoremus — CHANGES_REQUESTED** [review]
As stated inline, locationState needs to be updated to dispatch an Action to a Redux reducer via an action creator.

File: src/redux/remoteData/asyncActionCreator.ts (hunk 1 above)
Comment (nickoneill) [reply src/redux/remoteData/asyncActionCreator.ts]: It didn't look like `setState` is available here. How would I go about getting that in this context?

Comment (cdoremus) [reply src/redux/remoteData/asyncActionCreator.ts]: Use the dispatch function to call the appropriate action creator.

Comment (nickoneill) [reply src/redux/remoteData/asyncActionCreator.ts]: Ohhhh right

**Comment by nickoneill** [comment]
OK! Thanks for pointing that out, makes total sense.

**Review by cdoremus — APPROVED** [review]
Thanks for making the changes.

**Pull request merged**